custom_components/generic_hygrostat/binary_sensor.py

```diff
@@ -184,12 +184,14 @@ def get_lowest_sample(self):
             return None
 
     def set_dehumidification_target(self):
-        """Setting dehumidification target to min humidity sample + offset."""
+        """Setting dehumidification target to sum of min humidity sample and offset."""
+        self.combined_offset_tuple = (min(self.samples), self.target_offset)
+        self.combined_offset = sum(self.combined_offset_tuple)
         if self.target is None:
```

Comment:
I don't want to be rude, but I really don't like the code:
- Why `self.`? These variables are only used within this function right?
- Also, they're never used if `self.target` is not `None`, so they should be within the statement.
- Why putting these two different variables together in the same collection/tuple?
- And finally the don't get the variable name `combined_offset`? 

Sry, I'll close this PR for now.